litmus-portal/cluster-agents/subscriber/pkg/requests/webhook.go

```diff
@@ -128,12 +128,12 @@ func RequestProcessor(clusterData map[string]string, r types.RawData) error {
 		logrus.Print("LOG REQUEST ", r.Payload.Data.ClusterConnect.Action.ExternalData)
 		k8s.SendPodLogs(clusterData, podRequest)
 	} else if strings.Index("create update delete get", strings.ToLower(r.Payload.Data.ClusterConnect.Action.RequestType)) >= 0 {
-		_, err := k8s.ClusterOperations(r.Payload.Data.ClusterConnect.Action.K8SManifest, r.Payload.Data.ClusterConnect.Action.RequestType, r.Payload.Data.ClusterConnect.Action.Namespace)
+		_, err := k8s.ClusterOperations(r.Payload.Data.ClusterConnect.Action.K8SManifest, r.Payload.Data.ClusterConnect.Action.RequestType, r.Payload.Data.ClusterConnect.Action.Namespace, r.Payload.Data.ClusterConnect.Action.Username)
```

Comment:
Here could be changed to only pass `r.Payload.Data.ClusterConnect.Action` instead of all the current arguments.